.github/workflows/go_ci.yaml

```
@@ -0,0 +1,38 @@
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: CI (Go)
+
+on:
+  pull_request:
+    branches: ["main", "development"]
+  push:
+    branches: ["main", "development"]
+    
+jobs:
+ build:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 1

(indentation)